src/index.html

```diff
@@ -10,7 +10,28 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class ="search-form search-form_big" data-qa="big">
```

Comment:
This form should have just `class = search-form`, same for label `class ="search-form__label` and input `class= "search-form__input`

For small one you just add a modifier class `<form class ="search-form search-form--small"`
For small form label `<label class ="search-form__label search-form__label--small">`
For small form input `class ="search-form__input search-form__input--small"`